CMakeLists.txt

```diff
@@ -1,30 +1,42 @@
 cmake_minimum_required(VERSION 3.16)
-project(fixed_containers)
+#if(DEFINED ENV{VCPKG_ROOT} AND NOT DEFINED CMAKE_TOOLCHAIN_FILE)
+#  set(CMAKE_TOOLCHAIN_FILE "$ENV{VCPKG_ROOT}/scripts/buildsystems/vcpkg.cmake" CACHE STRING "")
+#endif()
+#if(NOT DEFINED CMAKE_TOOLCHAIN_FILE)
+#  set(CMAKE_TOOLCHAIN_FILE "${CMAKE_SOURCE_DIR}/vcpkg/scripts/buildsystems/vcpkg.cmake" CACHE STRING "")
+#endif()
```

Comment:
Intentionally leaving for merging?
I don't mind leaving this in, but I was curious: do you typically checkout vcpkg inside a repo root?

Reply:
I just to have this as a git submodule in my projects.